code-migration-detector/README.md

```diff
@@ -0,0 +1,98 @@
+# Code Migration Detector
```

Comment:
It seems there are no unit tests for the actual functionality in place (assertions that the test framework does its job properly)

Basically, this could be done by copying over the tests from the example and make assertions on the failing tests.

The reason is that the profile is not active on CI and will produce test failues (which is very nice do demo), but we need unit tests to assert this behaviour in CI.

Reply:
Do you now how to assert something about archunit test? 
I see two options:
- Run it like a maven integration test run and assert that the project setup fails with correct errors
- Learn how ArchUnit themselves test that their tests are running or failing and do the same here.

I'm for the option two... Will try... Stay tuned.